test/functional/apps/discover/group1/_inspector.ts

```diff
@@ -51,9 +49,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('should display request stats with no results', async () => {
       await inspector.open();
-      await testSubjects.click('inspectorRequestChooser');
       let foundZero = false;
-      for (const subj of ['Documents', 'Chart_data']) {
+      for (const subj of ['Documents', 'Data']) {
```

Comment:
Btw "Total hits" is not always present. Is it expected?

Reply:
The `Data` request is now coming from Lens and I'm not sure we can control its name, but I'll look into it.

It's expected to see `Data` when the chart is shown, and `Total hits` when the chart is hidden, but not both... Do you know under what circumstances you're seeing all three? It seems like it could be a bug.

Shown:
<img width="679" alt="Screen Shot 2022-11-28 at 12 34 41 PM" src="https://user-images.githubusercontent.com/25592674/204331872-627f66b8-aed5-46e1-ac73-96d7dffd0efe.png">

Hidden:
<img width="681" alt="Screen Shot 2022-11-28 at 12 35 16 PM" src="https://user-images.githubusercontent.com/25592674/204331893-e9c5c6bb-9e06-4854-b3c2-f579c0b822ff.png">

Reply:
> Do you know under what circumstances you're seeing all three? It seems like it could be a bug.

Open Discover with a data view which has a time field => Inspector will show Documents and Data. Switch to a data view without a time field => Inspector with show all 3 options. Reload the page => Documents and Total hits.

Reply:
Thanks! A generic "Visualization data" could work too.

Reply:
`Visualization data` would make sense to me, it's just not something we can update in this PR since it would affect many places that use expressions currently. I think we own this code now though, so it's something we could create an issue for.

We would either need to make a standard change in the data plugin to use `Visualization data`, or allow the title to be customized so it could be passed down: `Unified Histogram -> Lens -> Data Plugin`. Also worth noting this would need to be done for the `eql`, `esdsl`, and `essql` expression request handlers as well.